### PR TITLE
Update structure of the HtmlPageItemsConfig

### PR DESCRIPTION
**Pull request by bpauli**
- update structure of the HtmlPageItemsConfig

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: site/src/main/content/jcr_root/conf/aem-site-template-standard/_sling_configs/com.adobe.cq.wcm.core.components.config.HtmlPageItemsConfig/.content.xml [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+                        value="text/css"/>
+                </attributes>
+            </css>
+            <javascript
+                jcr:primaryType="nt:unstructured"
+                sling:configPropertyInherit="{Boolean}false"
+                sling:resourceType="nt:unstructured"
+                element="script"
+                location="header">
+                <attributes
```
Comment (vladbailescu) [inline site/src/main/content/jcr_root/conf/aem-site-template-standard/_sling_configs/com.adobe.cq.wcm.core.components.config.HtmlPageItemsConfig/.content.xml:48]: What about the `async` attribute?

File: site/src/main/content/jcr_root/conf/aem-site-template-standard/_sling_configs/com.adobe.cq.wcm.core.components.config.HtmlPageItemsConfig/.content.xml [hunk 2]
```diff
@@ -0,0 +1,98 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:sling="http://sling.apache.org/jcr/sling/1.0" xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0" xmlns:nt="http://www.jcp.org/jcr/nt/1.0"
+    jcr:primaryType="cq:Page">
+    <jcr:content
+        jcr:primaryType="cq:PageContent"
+        sling:configPropertyInherit="{Boolean}false"
+        prefixPath="/content/aem-site-template-standard.theme/_default">
+        <items
+            jcr:primaryType="nt:unstructured"
+            sling:configCollectionInherit="{Boolean}true"
```
Comment (vladbailescu) [inline site/src/main/content/jcr_root/conf/aem-site-template-standard/_sling_configs/com.adobe.cq.wcm.core.components.config.HtmlPageItemsConfig/.content.xml:10]: Why is `sling:configCollectionInherit` true and `sling:configPropertyInherit` false? Do we want to support inheritance of this configuration in the site tree?

File: site/src/main/content/jcr_root/conf/aem-site-template-standard/_sling_configs/com.adobe.cq.wcm.core.components.config.HtmlPageItemsConfig/.content.xml [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+            sling:resourceType="nt:unstructured">
```
Comment (vladbailescu) [inline site/src/main/content/jcr_root/conf/aem-site-template-standard/_sling_configs/com.adobe.cq.wcm.core.components.config.HtmlPageItemsConfig/.content.xml:11]: Is the `sling:resourceType` necessary? Will it not default to the primary type anyway?

**Review by vladbailescu — CHANGES_REQUESTED** [review]

File: site/src/main/content/jcr_root/conf/aem-site-template-standard/_sling_configs/com.adobe.cq.wcm.core.components.config.HtmlPageItemsConfig/.content.xml (hunk 1 above)
Comment (bpauli) [reply site/src/main/content/jcr_root/conf/aem-site-template-standard/_sling_configs/com.adobe.cq.wcm.core.components.config.HtmlPageItemsConfig/.content.xml:53]: added in f9e8472ad64dedf84184babaeeb196e9f073df26

File: site/src/main/content/jcr_root/conf/aem-site-template-standard/_sling_configs/com.adobe.cq.wcm.core.components.config.HtmlPageItemsConfig/.content.xml (hunk 3 above)
Comment (bpauli) [reply site/src/main/content/jcr_root/conf/aem-site-template-standard/_sling_configs/com.adobe.cq.wcm.core.components.config.HtmlPageItemsConfig/.content.xml:11]: removed in f9e8472ad64dedf84184babaeeb196e9f073df26. Was added by the CA Config Editor

File: site/src/main/content/jcr_root/conf/aem-site-template-standard/_sling_configs/com.adobe.cq.wcm.core.components.config.HtmlPageItemsConfig/.content.xml (hunk 2 above)
Comment (bpauli) [reply site/src/main/content/jcr_root/conf/aem-site-template-standard/_sling_configs/com.adobe.cq.wcm.core.components.config.HtmlPageItemsConfig/.content.xml:10]: removed in f9e8472ad64dedf84184babaeeb196e9f073df26. Was added by the CA Config Editor

File: site/src/main/content/jcr_root/conf/aem-site-template-standard/_sling_configs/com.adobe.cq.wcm.core.components.config.HtmlPageItemsConfig/.content.xml (hunk 1 above)
Comment (vladbailescu) [reply site/src/main/content/jcr_root/conf/aem-site-template-standard/_sling_configs/com.adobe.cq.wcm.core.components.config.HtmlPageItemsConfig/.content.xml:46]: It should be on the JS link though, shouldn't it?

File: site/src/main/content/jcr_root/conf/aem-site-template-standard/_sling_configs/com.adobe.cq.wcm.core.components.config.HtmlPageItemsConfig/.content.xml [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+                <attributes
+                    jcr:primaryType="nt:unstructured">
+                    <href
+                        jcr:primaryType="nt:unstructured"
+                        name="href"
+                        value="/css/theme.css"/>
+                    <rel
+                        jcr:primaryType="nt:unstructured"
+                        name="rel"
+                        value="stylesheet"/>
```
Comment (jckautzmann) [inline site/src/main/content/jcr_root/conf/aem-site-template-standard/_sling_configs/com.adobe.cq.wcm.core.components.config.HtmlPageItemsConfig/.content.xml:22]: Shouldn't it be: "preload stylesheet"?

Comment (jckautzmann) [reply site/src/main/content/jcr_root/conf/aem-site-template-standard/_sling_configs/com.adobe.cq.wcm.core.components.config.HtmlPageItemsConfig/.content.xml]: addressed in https://github.com/adobe/aem-site-template-standard/pull/71/commits/2e980e1ff80adef838ee0e4a4c7a40956bf40c55

**Review by vladbailescu — APPROVED** [review]

**Pull request merged**